lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -146,6 +150,29 @@ export abstract class BaseManagedIdentitySource {
                 managedIdentityId
             );
 
+        if (managedIdentityRequest.revokedTokenSha256Hash) {
```

Comment:
should this condition check for `revokedTokenSha256Hash` or `claims`

Reply:
It's both at this point in the code. `revokedTokenSha256Hash` only exists if claims exist, and then if there is a cached token and if the Managed Identity source supports token revocation